xclim/testing/tests/test_calendar.py

```diff
@@ -147,6 +147,51 @@ def test_adjust_doy_360_to_366():
     assert out.sel(dayofyear=366) == source.sel(dayofyear=360)
 
 
+def test_adjust_doy__max_93_to_max_94():
+    # GIVEN
+    source = xr.DataArray(np.arange(92), coords=[np.arange(152, 244)], dims="dayofyear")
+    time = xr.cftime_range("2000-06-01", periods=92, freq="D", calendar="all_leap")
+    target = xr.DataArray(np.arange(len(time)), coords=[time], dims="time")
+    # WHEN
+    out = adjust_doy_calendar(source, target)
+    # THEN
+    assert out[0].dayofyear == 153
+    assert out[0] == source[0]
+    assert out[-1].dayofyear == 244
+    assert out[-1] == source[-1]
+
+
+# def test_adjust_doy__leap_to_noleap_djf():
+#     # FIXME:
```

Comment:
I'm unsure how to handle partial time-series in-between two years, such as DJF case.